scenes/game_elements/characters/player/components/player.gd

```diff
@@ -12,18 +13,25 @@ enum Mode {
 	FIGHTING,
 }
 
+const DEFAULT_SPRITE_FRAME: SpriteFrames = preload(
+	"res://scenes/game_elements/characters/shared_components/sprite_frames/story_weaver.tres"
+)
+
 ## Controls how the player can interact with the world around them.
 @export var mode: Mode = Mode.COZY:
 	set = _set_mode
 @export_range(10, 100000, 10) var walk_speed: float = 300.0
 @export_range(10, 100000, 10) var run_speed: float = 500.0
 @export_range(10, 100000, 10) var stopping_step: float = 1500.0
 @export_range(10, 100000, 10) var moving_step: float = 4000.0
+@export var sprite_frames: SpriteFrames = DEFAULT_SPRITE_FRAME:
+	set = _set_sprite_frames
```

Comment:
I wonder if a subset of the commit message could become a docstring here:


```
The SpriteFrames must have the following animation names with the following number of frames:

- idle: 10 frames
- walk: 6 frames
- attack_01: 4 frames
- attack_02: 4 frames
```

Reply:
Good point, addressed. I think that we'll also need user documentation for creating your own player assets. I count with the Learning team for this.

Reply:
Agreed!